src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/eye.hpp

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+
+#include "shared_test_classes/base/ov_subgraph.hpp"
+
+namespace ov {
+namespace test {
+using EyeLayerTestParams = std::tuple<
+    std::vector<ov::Shape>,  // eye shape
```

Comment:
Use `vector::InputShape` instead to support dynamic case

Reply:
This test model doesn't have parameter nodes at all. It uses constants as inputs. So we don't need dynamic cases here